src/bridge/ui_commands.rs

```diff
@@ -129,108 +138,130 @@ pub enum ParallelCommand {
     },
 }
 
+async fn display_awailable_fonts(
```

Comment:
Not sure if this is introduced by this PR, but — typo.

```suggestion
async fn display_available_fonts(
```

Reply:
fixed